api/routes.py

```diff
@@ -366,6 +368,10 @@ def track_analytics_event(identifier_type, identifier, event_type):
 @requires_auth
 @returns_problem_detail
 def adobe_vendor_id_get_token():
+    if not app.manager.adobe_vendor_id:
+        return REMOTE_INTEGRATION_FAILED.detailed(
+            _("This server does not have an Adobe Vendor ID server configured.")
+        )
```

Comment:
This is an unrelated change to replace a 500 error I found when getting Open Ebooks QA working with a more helpful error. REMOTE_INTEGRATION_FAILED isn't quite right but it's close enough, I think.